modules/optimization/doc/content/modules/optimization/getting_started/InvOptTheory.md

```diff
@@ -0,0 +1,376 @@
+- [Parameter Derivatives of PDE:](#sec:PDEDerivs) Derivatives of the PDE with respect to the optimization parameters for force and material inversion.  Force inversion examples are provided where PDE constrained optimization is used to parameterize boundary conditions and body loads is provided on this [section](#sec:forceInvExample).  Material inversion examples are provided on this [section](#sec:forceInvExample) where PDE constrained optimization is used to identify material properties.
+
+The overall flow of the optimization algorithm implemented in the MOOSE optimization module is shown in [fig:optCycle].  In this example, the internal heat source, $q_v$, is being parameterized to match the simulated and experimental steady state temperature fields, $\widetilde{T}$ and $T$, respectively.  Step one of the optimization cycle consists of adjusting the internal heat source, $q_v$.  In step two, the physics model is solved with the current $q_v$ to obtain a simulated temperature field, $T$.  Step two is often referred to as the forward solve.  In step three, the simulated and experimental temperature fields are compared via the objective function, $J$.  If $J$ is below the user defined threshold, the optimization cycle stops and the best fit parameterization of $q_v$ is found.  If $J$ is above the user defined threshold, the optimization algorithm determines a new $q_v$ and the process is repeated.  In the next section, methods for determining the next iteration of the parameterized value, in this case $q_v$, will be presented.  Problems solved using the optimization module are available on the [Examples page](examples/index.md).
+
+!media media/fig_optCycle.png
+       style=width:80%;margin:auto;padding-top:2.5%;
+       id=fig:optCycle
+       caption=Optimization cycle example for parameterizaing an internal heat source distribution $q_v$ to match the simulated and experimental temperature field, $T$ and $\widetilde{T}$, respectively.
+
+
+
+[comment0]: <> (% ----------------------------------------------------------------------------------------------------------%)
+
+## PDE Constrained Inverse Optimization id=sec:invOptTheory
+
+Inverse optimization is a mathematical framework to infer model parameters by minimizing the misfit between the experimental and simulation observables.  In the optimization module, our model is a PDE describing the physics of the experiment.  We solve our physics PDE using the finite element method as implemented in MOOSE.  The physics of our problem constrains our optimization algorithm.  A PDE-constrained inverse optimization framework is formulated as an abstract optimization problem [!citet](biegler2003large):
+\begin{equation}\label{eq:optimization}
+   \min_{\mathbf{p}} J\left(\mathbf{u},\mathbf{p}\right);\quad~\textrm{subject to}~\mathbf{g}\left(\mathbf{u},\mathbf{p}\right)=\mathbf{0} ,
```

Comment:
Suggest to change \mathbf{g} to \mathbf{r} for showing that it is the residual of the PDE.

Reply:
Oh yea I think I made a similar comment to @lynnmunday at one point

Reply:
done, I used \mathbf{R}